src/plugins/application_config/server/routes/index.ts

```diff
@@ -85,8 +85,12 @@ export async function handleGetEntityConfig(
   response: OpenSearchDashboardsResponseFactory,
   logger: Logger
 ) {
+  logger.info(`Received a request to get entity config for ${request.params.entity}.`);
```

Comment:
should we keep this logging?

Reply:
I intentionally added it for audit purposes.

Reply:
I know we have telemetry stuff in the works, where you thinking of audit purposes in terms of usage collection or just how people utilize it?

Reply:
One case is for ease of troubleshooting why the configuration values are changed.